dts/arm/infineon/cat1b/cyw20829/cyw20829.dtsi

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m33";
+			reg = <0>;
+		};
+	};
+
+	flash-controller@40250000 {
+		compatible = "infineon,cat1-flash-controller";
+		reg = < 0x40250000 0x10000 >;
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+	flash0: flash@60002600{
```

Comment:
This is an external flash and should probably be in the board's DT

Reply:
moved the flash node into the board's dtsi files..